src/script/src/python/engine.rs

```diff
@@ -439,8 +439,8 @@ from greptime import col
 
 @copr(args=["number"], returns = ["number"], sql = "select * from numbers")
 def test(number) -> vector[u32]:
-    from greptime import dataframe
-    return dataframe().filter(col("number")==col("number")).collect()[0][0]
+    from greptime import PyDataFrame
```

Comment:
Do we need to update doc for this change?

Reply:
Yes, will update soon

> Do we need to update doc for this change?